src/features/profile/screens/profile.screen.js

```diff
@@ -0,0 +1,169 @@
+import React, { useContext } from "react";
+import { TouchableOpacity, FlatList } from "react-native";
+import { Avatar, Button, IconButton, Surface } from "react-native-paper";
+import styled from "styled-components/native";
+
+import { ProfileTabs } from "../components/profile-tabs.navigator";
+
+import { SafeArea } from "../../../components/utilities/safe-area.components";
+import { Text } from "../../../components/typography/text.components";
+import { colors } from "../../../infrastructure/theme/colors";
+import { AuthenticationContext } from "../../../services/authentication/authentication.context";
+
+import userProfile from "../../../utils/mock/userProfile";
+
+const SettingsButton = styled.View`
+  padding: ${(props) => props.theme.space[3]};
+  align-items: flex-end;
+`;
+
+const Name = styled(Text)`
+  font-family: ${(props) => props.theme.fonts.body_medium};
+  font-size: ${(props) => props.theme.fontSizes.h4};
+  padding: ${(props) => props.theme.space[2]};
+`;
+
+const FollowsSection = styled.View`
+  padding: ${(props) => props.theme.space[2]};
+  flex-direction: row;
+  align-items: center;
+`;
+
+const FollowNumbersFont = styled(Text)`
+  font-size: ${(props) => props.theme.fontSizes.h5};
+  font-family: ${(props) => props.theme.fonts.body_bold};
+`;
+
+const FollowTextFont = styled(Text)`
+  font-size: ${(props) => props.theme.fontSizes.body};
+  font-family: ${(props) => props.theme.fonts.heading};
+`;
+
+const SliceText = styled(Text)`
+  padding-right: ${(props) => props.theme.space[3]};
+  padding-left: ${(props) => props.theme.space[3]};
+`;
+
+const EditProfileButton = styled(Button).attrs({
+  labelStyle: {
+    color: colors.brand.primary,
+    fontSize: 20,
+  },
+})`
+  border-width: 1px;
+  border-radius: 20px;
+  border-color: ${(props) => props.theme.colors.brand.primary};
+`;
+
+const EditProfileButtonContainer = styled.View`
+  padding: ${(props) => props.theme.space[3]};
+  width: 70%;
+`;
+
+const ProfileInfoContainer = styled.View`
+  align-items: center;
+`;
+
+export const ProfileScreen = ({ navigation }) => {
+  const { user } = useContext(AuthenticationContext);
+  // const [photo, setPhoto] = useState(null);
+  // const [fullName, setFullName] = useState("");
+
+  // const getProfileInfo = async (currentUser) => {
+  //   const photoUri = await AsyncStorage.getItem(`${currentUser.uid}-photo`);
+  //   if (!photoUri) {
+  //     setPhoto(currentUser.photoURL + "?type=large");
+  //     setFullName(currentUser.displayName)
+  //   } else {
+  //     setPhoto(photoUri);
+  //   }
+  // };
+
+  // useFocusEffect(
+  //   useCallback(() => {
+  //     getProfileInfo(user);
+  //   }, [user])
+  // );
```

Comment:
Is there a reason not to delete these lines of code too?

Reply:
I might use this when firestore is implemented.